src/webgl/material.js

```diff
@@ -191,17 +191,7 @@ p5.prototype.createShader = function(vertSrc, fragSrc) {
  * The fragment shader is given a uniform, or variable, called `tex0`.
  * This is created specificially for filter shaders to access the canvas contents.
```

Comment:
Maybe change the language slightly here to say that this function is intended to be used with the `filter()` function. 

"createFilterShader is intended to be used in tandem with the <a href="filter function...">`filter()`</a> for filtering the entire contents of a canvas when using webGL mode.